Bot Sharder/README.md

```diff
@@ -1,112 +1,35 @@
-  "actions": [
-    {
-      "behavior": "1",
-      "interpretation": "0",
-      "code": "client.shard && client.shard.fetchClientValues('users.size')\n.then(results => {\n    var output = results.reduce(function(accumulator, currentValue){\n      return accumulator + currentValue;\n    });\n    this.storeValue(output, 1 ,\"totalUsers\", cache)\n    this.callNextAction(cache)\n}).catch(console.error);",
-      "storage": "0",
-      "varName": "",
-      "name": "Run Script"
-    },
-    {
-      "behavior": "1",
-      "interpretation": "0",
-      "code": "client.shard && client.shard.fetchClientValues('guilds.size')\n.then(results => {\n    var output = results.reduce(function(accumulator, currentValue){\n      return accumulator + currentValue;\n    });\n    this.storeValue(output, 1 ,\"totalGuilds\", cache)\n    this.callNextAction(cache)\n}).catch(console.error);",
-      "storage": "0",
-      "varName": "",
-      "name": "Run Script"
-    },
-    {
-      "channel": "0",
-      "varName": "",
-      "message": "There are a total of ${tempVars(\"totalUsers\")} users in \n${tempVars(\"totalGuilds\")} guilds across all shards.\n\nThis command was ran in shard ${(client.shard.id + 1)} and has an id \nof ${(client.shard.id)}.\n\n\nTotal Active Shards: ${client.shard.count}\n",
-      "storage": "0",
-      "varName2": "",
-      "name": "Send Message"
-    }
-  ]
-}
-```
-
-
-Run Scripts
-====
-
-  The command above uses these scripts.
-
-*Make sure **End Behavior** is set to **Do Not Call Next Action**, and **Evaluate Text First** is selected for interpretation  style for all of these scripts*
-----------------------------------------------------------------------------------------------------------------------------------------
-**Total Amount Of Users**
-
-This script will store all users to a temp variable called totalUsers, `There are a total of ${tempVars("totalUsers")} users`
-```js
-client.shard.fetchClientValues('users.size')
-.then(results => {
-    var output = results.reduce(function(accumulator, currentValue){
-      return accumulator + currentValue;
-    });
-    this.storeValue(output, 1 ,"totalUsers", cache)
-    this.callNextAction(cache)
-}).catch(console.error);
-```
-
-**Total Amount Of Guilds**
-
-This script will store all users to a temp variable called totalGuilds, `users in ${tempVars("totalGuilds")} guilds`
-```js
-client.shard.fetchClientValues('guilds.size')
-.then(results => {
-    var output = results.reduce(function(accumulator, currentValue){
-      return accumulator + currentValue;
-    });
-    this.storeValue(output, 1 ,"totalGuilds", cache)
-    this.callNextAction(cache)
-}).catch(console.error);
-```
+If you want to run it with a linux terminal then instead of running it with `bot node.js` you run it `node sharded-bot.js` if you want to provide a different shard count its `node sharded-bot.js shard_count=3` if you wanted 3 shards ( This is usually not needed unless your bot is in 5000 guilds.  )
```

Comment:
`bot node.js` huh? Besides, this phrase could use some rewording

Reply:
Didn't see that, was wrex's doings. Just left it